src/server/utils/cassandra.js

```diff
@@ -1,10 +1,11 @@
 import Promise from 'bluebird';
 import config from 'config';
-import { Client, types as CassandraTypes } from 'cassandra-driver';
+import { Client, types as CassandraTypes } from 'dse-driver';
 import { lookupServiceAsync } from './lookup-service';
 
 // Client promises by keyspace
 const clientPromises = new Map();
+const dse = require('dse-driver');
```

Comment:
Yea, now that I corrected the import statement all of the other "dse" related comments you made below fixed right up.  Makes sense.